package.json

```diff
@@ -66,9 +66,11 @@
     "@ionic/pwa-elements": "^3.2.2",
     "@sentry/angular-ivy": "^7.102.1",
     "@supabase/supabase-js": "^2.39.0",
+    "@types/chroma-js": "^2.4.4",
     "@types/file-saver": "^2.0.7",
     "bootstrap-datepicker": "^1.10.0",
     "capacitor-blob-writer": "^1.1.14",
+    "chroma-js": "^2.6.0",
```

Comment:
nit(non-blocking): interesting package. I think still probably better to use our mixins where possible to generate CSS at compile time, but definitely should unlock some interesting runtime colour functionality.